docs/HACKING.md

```diff
@@ -190,3 +190,11 @@ will attempt to clone them to a temporary directory.
 
 Bumping the release and manifest versions typically require updating the `elasticsearch-operator` as well.
 * `dep ensure -update github.com/openshift/elasticsearch-operator`
+
+## Deploying without OLM
+Production relies upon OLM to manage and control the operator deployment, permissions, etc. The manifest defines all the resources needed by OLM.  We can use this same manifest to generate a list of resources to deploy without using OLM.
+```
+make deploy-image                                                                                                                                                                                                                                                                               
+
+./hack/gen-olm-artifacts.sh manifests/5.1/cluster-logging.v5.1.0.clusterserviceversion.yaml  $CLO_IMAGE | oc create -f -
```

Comment:
Can you describe, for testing purposes, to what we should point the CLO_IMAGE to?